build.ps1

```diff
@@ -122,13 +122,14 @@ Get-ChildItem $ReferenceDocset -Directory -Exclude 'docs-conceptual', 'mapping',
         }
         catch {
             $allErrors += $_
+            $_
```

Comment:
```suggestion
            Write-Error -Message "PlatyPS failure: $ModuleName -- $Version" -Exception $_
```

If you make this Write-Error it will be much easier to find.